test/dropbox/docker-compose.yml

```diff
@@ -3,13 +3,9 @@ version: '3'
 services:
   openapi_mock:
     image: muonsoft/openapi-mock
-    volumes:
-      - dropbox_backup_data:/var/lib/dav
-      - ./user_v2.yaml:/etc/openapi/user_v2.yaml
-      - ./openapi-mock.yaml:/etc/openapi/openapi-mock.yaml
     environment:
-      - OPENAPI_MOCK_USE_EXAMPLES: if_present
-      - OPENAPI_MOCK_SPECIFICATION_URL: https://raw.githubusercontent.com/offen/docker-volume-backup/2ea59a649be79d6cb22b8b5ad29e24377d4ab9d4/test/dropbox/user_v2.yaml
+      OPENAPI_MOCK_USE_EXAMPLES: if_present
+      OPENAPI_MOCK_SPECIFICATION_URL: https://raw.githubusercontent.com/offen/docker-volume-backup/2ea59a649be79d6cb22b8b5ad29e24377d4ab9d4/test/dropbox/user_v2.yaml
```

Comment:
Don't know how to bring the spec file into the CI... when it's part of the master branch, this needs to updated (points to this PR right now to get the CI working).

Reply:
If I understand the documentation correctly you can:

a. mount the spec file into the container like `./user_v2.yml:/etc/openapi/user_v2.yml`
b. pass the path here (it's odd the param is named `_URL`, yes): `OPENAPI_MOCK_SPECIFICATION_URL: '/etc/openapi/user_v2.yaml'`

N.B.: I did not test this, it's just from the README here: https://github.com/muonsoft/openapi-mock

Reply:
I had it like that before, but I thought the CI only has the docker image and therefore no access to the repo source code.
If it has access, do you know where its scope is? Is it in the root of the project or in the respective test subfolder already? CI debugging is so time consuming unfortunately, maybe I need to install a local action runner to debug this faster.

Reply:
In the compose file, . should resolve to the directory of the test itself.